Mesh_2/doc/Mesh_2/CGAL/IO/vtk_io_2.h

```diff
@@ -0,0 +1,19 @@
+namespace CGAL{
+//!\ingroup PkgMesh2
+//! \brief writes the content of a `CDT` in the .vtu
+//! XML format. 
+//! 
+//! The triangles inside the domain and the constrained edges will be outputted.
+//! 
+//! \tparam CDT a `Constrained_Delaunay_triangulation_2`.
+//! 
+//! \param os a `std::ostream`.
+//! \param tr an instance of `CDT` to be written.
+//! \param binary decides if the data should be written in binary(`true`)
+//!   or in ASCII(`false`).
+//!
+template <class CDT>
+void write_VTU_2D(std::ostream& os,
```

Comment:
I don't like this name, worse case to respect CGAL conventions it should be `_2` but for free function we usually don't have dimension suffix.

Reply:
+ `changes.md` has not been updated.